### PR TITLE
Notify unlinked user about empty LASN

### PR DESCRIPTION
**Pull request by lnogueir**
As pointed out by @jeverd, after adding `Unlink` endpoint, it is now possible for a user to unlink and stay subscribed. In that case, we should still notify him that his LASN is now empty.

### CODE REVIEW

File: services/lasn/lasn.go [hunk 1]
```diff
@@ -112,18 +123,18 @@ func (s *Service) Subscribe(request *protos.SubscribeRequest, stream protos.Lasn
 	return ctx.Err()
 }
 
-func (s *Service) notifyUsersWith(userSummaries []*protos.UserSummary) {
+func (s *Service) sendUpdatesTo(userIds []string, updatedUserSummaries []*protos.UserSummary) {
```
Comment (SergioSanchez12) [inline services/lasn/lasn.go:126]: Will probably remove this updatedUserSummaries argument when I merge this into the filtering PR. Other than that it I think it looks good

**Review by SergioSanchez12 — APPROVED** [review]

File: services/lasn/lasn.go (hunk 1 above)
Comment (lnogueir) [reply services/lasn/lasn.go:126]: Wait, so how you know what user summaries to send?

Comment (SergioSanchez12) [reply services/lasn/lasn.go:126]: I will call `RetrieveFilterUsersConnectedTo` on each ID in `userIds` array

Comment (lnogueir) [reply services/lasn/lasn.go:126]: Ah, fair enough. That's better I think.

**Pull request merged**